diff --git a/gdraw/drawboxborder.c b/gdraw/drawboxborder.c
index 629968608..0b00e9dd5 100644
--- a/gdraw/drawboxborder.c
+++ b/gdraw/drawboxborder.c
@@ -24,6 +24,7 @@
  * OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+#include <fontforge-config.h>
 #include <math.h>
 #include <gdraw.h>
 #include <ggadget.h>
diff --git a/gdraw/gpsdraw.c b/gdraw/gpsdraw.c
index 5e58b6208..099e41bbd 100644
--- a/gdraw/gpsdraw.c
+++ b/gdraw/gpsdraw.c
@@ -24,6 +24,7 @@
  * OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+#include "fontforge-config.h"
 #include <stdio.h>
 #include <math.h>
 #include <time.h>
@@ -31,7 +32,6 @@
 #if !defined(__MINGW32__)
 #include <sys/wait.h>
 #endif
-#include "fontforge-config.h"
 #include <unistd.h>
 
 #include "gpsdrawP.h"
